client/src/components/search/searchBar/searchbar-base.css

```diff
@@ -617,10 +617,13 @@ a[class^='ais-'] {
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
-  position: absolute;
   z-index: 100;
   width: 20px;
   height: 20px;
+}
+.ais-SearchBox-reset,
+.ais-SearchBox-loadingIndicator {
```

Comment:
These two are not displayed.
Should we just display none them and remove their additional styles?

Reply:
one of them has display none in the `searchbar.css` file, and in `search-bar.tsx` the `showLoadingIndicator` set to `false`

```css
// search-bar.css
.ais-SearchBox-reset {
  display: none;
}
```

```ts
// search-bar.tsx
<SearchBox
                    focusShortcuts={['83', '191']}
                    onChange={this.handleChange}
                    onSubmit={e => {
                      this.handleSearch(e);
                    }}
                    showLoadingIndicator={false}
                    translations={{ placeholder }}
                  />
 ```

I wanted to delete them entirely, because we aren't using them. I happy with either

Reply:
I can set their values to `display: none` in the attribute and open 2nd PR for the deletion, if we wanted delete them.

Reply:
Sure both make sense.